database/upgrade_scripts/028-organization.sql

```diff
@@ -0,0 +1,38 @@
+-- -----------------------------------------------------
+-- Table vmaas.organization
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS organization (
+  id SERIAL,
+  name TEXT NOT NULL UNIQUE, CHECK (NOT empty(name)),
+  PRIMARY KEY (id)
+)TABLESPACE pg_default;
+
+INSERT INTO organization (id, name) VALUES
+  (1, 'DEFAULT');
+
+-- -----------------------------------------------------
+-- vmaas users permission setup:
+-- vmaas_writer - has rights to INSERT/UPDATE/DELETE; used by reposcan
+-- vmaas_reader - has SELECT only; used by webapp
+-- -----------------------------------------------------
+GRANT SELECT, INSERT, UPDATE, DELETE ON TABLE public.organization TO vmaas_writer;
+GRANT USAGE, SELECT, UPDATE ON SEQUENCE public.organization_id_seq TO vmaas_writer;
+GRANT SELECT ON TABLE public.organization TO vmaas_reader;
+
+ALTER TABLE repo
+  ADD COLUMN org_id INT NULL,
```

Comment:
`org_id` might be a bit unfortunate name for this, as this might be confused with org_id on SaaS as opposed to Satellite's internal org id.

Would an integer be enough?